level_test.go

```diff
@@ -21,11 +21,25 @@
 package zap
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+func TestLevelEnablerFunc(t *testing.T) {
+	//testing for the level enabler, I will generate a logger with the given
```

Comment:
None of the other tests have comments like these; we should either brush up the grammar and formatting, or drop it (my preference is for the latter).